### PR TITLE
add annotations to fix scorecard test failures

### PR DESCRIPTION
**Pull request by nishant-parekh**
Signed-off-by: Nishant Parekh <nparekh@redhat.com>

### CODE REVIEW

**Review by donpenney — APPROVED** [review]
/lgtm

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -49,5 +49,4 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
-      serviceAccountName: controller-manager
```
Comment (nishant-parekh) [inline config/manager/manager.yaml:52]: this line @irinamihai @jc-rh

Comment (irinamihai) [reply config/manager/manager.yaml:52]: @rcarrillocruz @vitus133

**Review by donpenney — COMMENTED** [review]
/lgtm

**Pull request merged**